scripts/pin_the_slow_way.py

```diff
@@ -29,11 +29,14 @@
 pinned = {
           'boost': 'boost 1.62.*',
           'bzip2': 'bzip2 1.0.*',
+          'cairo': 'cairo 1.14.*',
           'ffmpeg': 'ffmpeg 2.8.*',
-          'fontconfig': 'fontconfig 2.11.*',
+          'fontconfig': 'fontconfig 2.12.*',
           'freetype': 'freetype 2.6.*',
           'geos': 'geos 3.5.*',
           'giflib': 'giflib 5.1.*',
+          'glib': 'glib 2.51.*',
```

Comment:
There are several symbol additions in the 2.51 series of `glib`. Maybe it is time to look at a tighter pinning.

ref: https://abi-laboratory.pro/tracker/timeline/glib/